src/io/flutter/view/InspectorPanel.java

```diff
@@ -782,6 +804,10 @@ public void customizeCellRenderer(
         setIcon(icon);
       }
     }
+
+    private void appendSearch(@NotNull String text, @NotNull SimpleTextAttributes attributes) {
```

Comment:
nit: maybe just keep calling this append as it is still just appending text but now it is appending text in a way that is compatible with search.

Reply:
`append` is final in super but I agree so made it `appendText` 👍